schema/artwork.js

```diff
@@ -11,6 +15,21 @@ import {
   GraphQLInt
 } from 'graphql';
 
+export let ArtworkPredicates = {
+  is_contactable: (artwork, relatedSales) => {
+    return artwork.forsale && !_.isEmpty(artwork.partner) && !artwork.acquireable && !relatedSales.length
+  }
+}
+
+let fetchRelatedSales = ({ id }, options) => {
+  options = qs.stringify(_.defaults(options, {
+    size: 1,
+    active: true,
```

Comment:
@broskoski a question [from the future]:

What is this `active: true` attempting to do? The [endpoint](https://github.com/artsy/gravity/blob/800d34f8477794ca2d3bb8e37217644cba127c06/app/api/v1/related_endpoint.rb#L224) doesn't support that param, nor did it ever (from what I can tell). There's a `live: true` parameter which should maybe be substituted.